.goreleaser.yml

```diff
@@ -36,10 +36,47 @@ dockers:
 - goos: linux
   goarch: amd64
   image_templates:
-  - "percona/mongodb_exporter:{{.Major}}.{{.Minor}}"
-  - "percona/mongodb_exporter:{{.Version}}"
+  - "docker.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64"
+  - "docker.io/percona/mongodb_exporter:{{.Version}}-amd64"
+  - "ghcr.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64"
+  - "ghcr.io/percona/mongodb_exporter:{{.Version}}-amd64"
+  dockerfile: Dockerfile
+  build_flag_templates:
+    - "--pull"
+    - "--platform=linux/amd64"
+- goos: linux
+  goarch: arm64
+  image_templates:
+  - "docker.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8"
+  - "docker.io/percona/mongodb_exporter:{{.Version}}-arm64v8"
+  - "ghcr.io/percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8"
+  - "ghcr.io/percona/mongodb_exporter:{{.Version}}-arm64v8"
   dockerfile: Dockerfile
   use: buildx
```

Comment:
@nikita-b / @BupycHuk Do we need to use buildx for our Docker builds?

Reply:
We build this image only for the community and I think it's okey to use buildx :)